source/Ship.cpp

```diff
@@ -3126,6 +3126,15 @@ void Ship::Restore()
 
 
 
+bool Ship::IsDamaged() const
+{
+	// Account for ships with no shields when determining if they're damaged.
+	return (attributes.Get("shields")) == 0. ?
+		(Hull() != 1.) : (Shields() != 1. || Hull() != 1.);
```

Comment:
I believe this can be simplified to the following.
```suggestion
	return (attributes.Get("shields") != 0. && Shields() != 1.) || Hull() != 1.;
```